docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -3365,76 +3369,36 @@ List nodes
           },
           "Engine": {
             "EngineVersion": "1.12.0-dev",
+            "Labels": {
+                "foo": "bar",
+            }
             "Plugins": [
               {
                 "Type": "Volume",
                 "Name": "local"
               },
               {
                 "Type": "Network",
+                "Name": "bridge"
+              }
+              {
+                "Type": "Network",
+                "Name": "null"
+              }
+              {
+                "Type": "Network",
                 "Name": "overlay"
               }
             ]
           }
         },
         "Status": {
-          "State": "READY"
-        },
-        "Manager": {
-          "Raft": {
-            "RaftID": 10070664527094528000,
-            "Addr": "172.17.0.2:4500",
-            "Status": {
-              "Leader": true,
-              "Reachability": "REACHABLE"
-            }
-          }
+          "State": "ready"
         },
-        "Attachment": {
-          "Network": {
-            "ID": "4qvuz4ko70xaltuqbt8956gd1",
-            "Version": {
-              "Index": 6
-            },
-            "CreatedAt": "2016-06-07T20:31:11.912919752Z",
-            "UpdatedAt": "2016-06-07T20:31:11.921784144Z",
-            "Spec": {
-              "Name": "ingress",
-              "Labels": {
-                "com.docker.swarm.internal": "true"
-              },
-              "DriverConfiguration": {},
-              "IPAM": {
-                "Driver": {},
-                "Configs": [
-                  {
-                    "Family": "UNKNOWN",
-                    "Subnet": "10.255.0.0/16"
-                  }
-                ]
-              }
-            },
-            "DriverState": {
-              "Name": "overlay",
-              "Options": {
-                "com.docker.network.driver.overlay.vxlanid_list": "256"
-              }
-            },
-            "IPAM": {
-              "Driver": {
-                "Name": "default"
-              },
-              "Configs": [
-                {
-                  "Family": "UNKNOWN",
-                  "Subnet": "10.255.0.0/16"
-                }
-              ]
-            }
-          },
-          "Addresses": [
-            "10.255.0.2/16"
-          ]
+        "ManagerStatus": {
+          "Leader": true,
+          "Reachability": "reachable",
+          "Addr": "172.17.0.2:4500""
```

Comment:
@sfsmithcha noticed you're using the old `4500` ports here; can you change to `2377` ?